hack/generators/release-controllers/content/osd_rc_deployments.py

```diff
@@ -362,7 +362,7 @@ def _add_osd_rc_deployment(gendoc):
                                         ],
                             'image': 'release-controller-api:latest',
                             'name': 'controller',
-                            'volumeMounts': get_kubeconfig_volume_mounts(),
+                            'volumeMounts': get_rcapi_volume_mounts(),
```

Comment:
So, you can't have `volumeMounts` without a corresponding `volumes` definition...

Reply:
This is why the validation test failed ^^

Reply:
Fixed